crew_testing/tools/openapi_auth_detector.py

```
@@ -0,0 +1,109 @@
+                        "fixture": '@pytest.fixture\ndef auth_headers():\n    """Mock authentication headers for testing."""\n    return {"Authorization": "Bearer mock_token_for_testing"}',
+                        "usage": 'response = client.get("/api/matches", headers=auth_headers)',
+                    }
+                })
+            else:
+                return json.dumps({
+                    "requires_auth": False,
+                    "auth_type": None,
+                    "message": f"Endpoint {endpoint} does not require authentication",
+                })
+
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)